cmd/srcd/cmd/prune_test.go

```diff
@@ -0,0 +1,65 @@
+// +build integration
+
+package cmd
+
+import (
+	"context"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+
+	cmdtest "github.com/src-d/engine/cmd/test-utils"
+
+	"github.com/stretchr/testify/suite"
+)
+
+// This test suite does not test the `--with-images` flag.
+// If the tests are run with `make test-integration` the daemon image will be
+// the one build locally (e.g. srcd/cli-daemon:dev-b72f1fe), and deleting
+// this image would make all the other tests fail.
+
+type PruneTestSuite struct {
+	cmdtest.IntegrationSuite
+	testDir string
+}
+
+func TestPruneTestSuite(t *testing.T) {
+	s := PruneTestSuite{}
+	suite.Run(t, &s)
+}
+
+func (s *PruneTestSuite) SetupTest() {
+	var err error
+	s.testDir, err = ioutil.TempDir("", "prune-test")
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func (s *PruneTestSuite) TearDownTest() {
+	s.RunStop(context.Background())
+	os.RemoveAll(s.testDir)
+}
+
+func (s *PruneTestSuite) TestRunningContainers() {
+	require := s.Require()
+
+	_, err := s.RunInit(context.TODO(), s.testDir)
+	require.NoError(err)
+
+	_, err = s.RunSQL(context.TODO(), "SELECT 1")
+	require.NoError(err)
+
+	_, err = s.RunCommand(context.TODO(), "prune")
+	require.NoError(err)
+
+	s.AllStopped()
```

Comment:
for prune we should test not only that containers were stopped but that other stuff was removed (network, volumes)